components/breadcrumb/examples/10-overrides.js

```diff
@@ -56,14 +56,25 @@ function Example({ brand }) {
 			>
 				<Crumb href="#/" text="Home" />
 				<Crumb href="#/personal-banking/" text="Personal" />
-				<Crumb href="#/credit-cards/" text="Credit cards" />
+				<Crumb
+					overrides={{
+						CrumbRoot: {
```

Comment:
Maybe we should just call it `Root`?
I'm still not sure about this `[component]Root` thing...

Reply:
I would prefer Root but a component can have multiple Root components and overrides wouldnt work. Happy to hear suggesstions on naming, this was the nicest one I could think of at the time.

Reply:
Can they? Where? Maybe in a11y? But otherwise each component has a wrapping element no?

Reply:
In breadcrumb for example we export <Breadcrumb/> and <Crumb/> if both have the wrapping element named as <Root/> how would we do the overrides for them since we do overrides based on the package name i.e.
```
overridesWithTokens['@westpac/breadcrumb'] = {
  Root: {}, 
  Root: {}, 
}
```

Reply:
Fair point! Let's leave it as you had it

Reply:
@jonny found out that we can just use the component name as the root component (i.e. `<Breadcrumb/>` instead of `<BreadcrumbRoot/>`). Would there be any potential issues with declaring and using a variable with the same name as the function that contains it @dominikwilkowski ?